xtest/abac.py

```diff
@@ -623,14 +659,19 @@ def scs_create(self, scs: list[SubjectSet]) -> SubjectConditionSet:
         return SubjectConditionSet.model_validate_json(out)
 
     def scs_map(
-        self, sc: str | SubjectConditionSet, value: str | AttributeValue
+        self,
+        sc: str | SubjectConditionSet,
+        value: str | AttributeValue,
+        action: str | Action = "read",
     ) -> SubjectMapping:
         cmd: list[str] = self.otdfctl + "policy subject-mappings create".split()
 
         if self.flag_scs_map_action_standard:
-            cmd += ["--action-standard=read"]
+            cmd += [
+                f"--action-standard={action if isinstance(action, str) else action.name}"
+            ]
```

Comment:
This flag is deprecated in `otdfctl`. Can we add a comment about deprecation here? We probably don't want to change it at this second because of backwards compatibility tests relying on older clients, but policy CRUD doesn't have to be as backwards compatible as TDF interoperability.